src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource/help-buildForkPRMerge.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Whether to build pull requests filed from forks of the main repository.
+    The job will be named according to the PR and builds will attempt to merge with the base branch.
+</div>
```

Comment:
The latter will look like what is currently displayed, which is what I intended.